### ISSUE TITLE
Library not loaded: @rpath/libswiftSwiftOnoneSupport.dylib

### ISSUE THREAD

**Issue by jwelton** [net-a-porter-mobile/XCTest-Gherkin#152]
When integrating the library into a new project using Cocoapods, I'm getting the following runtime error:

```
The bundle “app-nameUITests” couldn’t be loaded because it is damaged or missing necessary resources. Try reinstalling the bundle.
2019-01-06 10:45:10.131902+0000 app-nameUITests-Runner[91337:12291379] (dlopen_preflight(/Users/Jake/Library/Developer/Xcode/DerivedData/app-name-baqtlkfrrpoabtgsnoxqtckmtaag/Build/Products/Debug-iphonesimulator/app-nameUITests-Runner.app/PlugIns/app-nameUITests.xctest/app-nameUITests): Library not loaded: @rpath/libswiftSwiftOnoneSupport.dylib
  Referenced from: /Users/Jake/Library/Developer/Xcode/DerivedData/app-name-baqtlkfrrpoabtgsnoxqtckmtaag/Build/Products/Debug-iphonesimulator/app-nameUITests-Runner.app/PlugIns/app-nameUITests.xctest/Frameworks/XCTest_Gherkin.framework/XCTest_Gherkin
  Reason: image not found)
```

This is my Podfile:
```
platform :ios, '12.1'

target 'my-app' do
  use_frameworks!

  target 'my-appTests' do
    inherit! :search_paths
  end

  target 'my-appUITests' do
    inherit! :search_paths

    pod 'XCTest-Gherkin/Native'
  end
end
```

I've had this issues when integrating into a few different projects, however, I cannot remember how I resolved the problem in the past. When I figure it out, I'll be sure to post here.

**Comment by deanWombourne** [comment]
Oh, sorry to hear you have trouble integrating our library! It should (in theory) just be a simple cocoa pods install, and as far as we know it doesn't do anything unusual which would cause runtime loader errors like this.

Do you have a trivial example project that we can take a look at - a zip upload, or a GitHub project for example?

**Comment by deanWombourne** [comment]
Oh, do you have "Always Embed Swift Standard Libraries" set to YES or NO in your build settings (for both the app and the test targets?)

**Comment by jwelton** [comment]
Sure. I've attached an example project that is giving me this error. Just document my steps:

- Create a new XCode project (10.01)
- For UITest target, set `Always Embed Swift Standard Libraries` to `default`. When creating a new project, XCode forces this to `YES`, which Cocoapods warns about when installing pods. The `default` setting for this is also YES, so in theory, this step doesn't change anything (I suspect it allows Cocoapods some control over it, but not 100% sure)
- Add Cocoapods ` Podfile`
- Run `pod install`
- Open workspace and try running the default UI test (no need to even import the XCTest-Gherkin module)

Regarding `Always Embed Swift Standard Libraries `. After following the above steps, this value is `NO` for the app target and `YES` for the UITest target. Forcing both of these to `YES` doesn't seem to resolve the issue.

I suspect this issue might be related to only adding the framework to the UITest module, but not 100% sure. I've actually managed to get things running on my projects, but unfortunately, I'm not 100% sure what steps I took to fix it.

I also suspect this might not be an issue related to this framework specifically, but wanted to open a ticket here so its documented incase anyone else comes across this problem.

[XCTest-Example.zip](https://github.com/net-a-porter-mobile/XCTest-Gherkin/files/2731918/XCTest-Example.zip)
